shell/browser/native_window_mac.h

```diff
@@ -86,6 +89,7 @@ class NativeWindowMac : public NativeWindow, public ui::NativeThemeObserver {
   bool IsExcludedFromShownWindowsMenu() override;
   void SetSimpleFullScreen(bool simple_fullscreen) override;
   bool IsSimpleFullScreen() override;
+  void UpdateFrame() override;
```

Comment:
Can you put the method in the same order with the base class? i.e. put it under the `RedrawTrafficLights` method, or move the declaration in base class to bellow `IsSimpleFullScreen`.